lib/commands/start.js

```diff
@@ -22,11 +22,18 @@ class StartCommand extends Command {
 
     run(argv) {
         const ProcessManager = require('../process-manager');
-        const chalk = require('chalk');
+        const getInstance = require('../utils/get-instance');
 
-        const instance = this.system.getInstance();
         const runOptions = {quiet: argv.quiet};
 
+        let instance;
+
+        try {
+            instance = getInstance(argv.name, this.system, 'start');
+        } catch (error) {
+            return Promise.reject(error);
```

Comment:
since we're not actually doing anything with the error here we can just make the above be:

```js
const instance = getInstance(argv.name, this.system, 'start');
```

any error generated by `getInstance` will get propagated fine, the `Promise.reject` wrapper is unnecessary.